cpp/modmesh/view/wrap_view.cpp

```diff
@@ -497,6 +497,12 @@ OneTimeInitializer<view_pymod_tag> & OneTimeInitializer<view_pymod_tag>::me()
 
 void initialize_view(pybind11::module & mod)
 {
+// TODO: There is some bug in QT RHI, this is a workaround to force QT use opengl directly
```

Comment:
Could you please add a reference (maybe URL) to where the QT RHI bug is described?

Reply:
fix in https://github.com/solvcon/modmesh/pull/195/commits/1727761ea359dd810c7be649c5128ef8e9310c1a

Reply:
Thanks